header/src/main/webapp/WEB-INF/ws-servlet.xml

```diff
@@ -15,10 +15,20 @@
         <property name="viewClass" value="org.springframework.web.servlet.view.JstlView"></property>
         <property name="prefix" value="/WEB-INF/jsp/"></property>
         <property name="suffix" value=".jsp"></property>
+        <property name="attributes">
+            <map>
+                <entry key="defaultLanguage" value="${language:en}" />
+                <entry key="consolePublicContextPath" value="${consolePublicContextPath:/console}" />
+            </map>
+        </property>
     </bean>
 
     <bean id="georchestraConfiguration" class="org.georchestra.commons.configuration.GeorchestraConfiguration">
         <constructor-arg value="header" />
     </bean>
 
+    <context:property-placeholder
+        location="file:${georchestra.datadir}/default.properties, file:${georchestra.datadir}/mapfishapp/mapfishapp.properties"
```

Comment:
you probably mean header.properties ;-)

Reply:
AHHHHH

Reply:
You're allowed to `git push --force` in the branch ;-)

Reply:
Yes. Done.